src/Kovan-DssSpell.t.sol

```diff
@@ -454,12 +452,12 @@ contract DssSpellTest is DSTest, DSMath {
 
     function checkCollateralValues(bytes32 ilk, SystemValues storage values) internal {
         (uint duty,)  = jug.ilks(ilk);
-        assertEq(duty, rates.rates(values.collaterals[ilk].pct / 10));
+        assertEq(duty, rates.rates(values.collaterals[ilk].pct));
         // make sure duty is less than 1000% APR
         // bc -l <<< 'scale=27; e( l(10.00)/(60 * 60 * 24 * 365) )'
         // 1000000073014496989316680335
         assertTrue(duty >= RAY && duty < 1000000073014496989316680335);  // gt 0 and lt 1000%
-        assertTrue(diffCalc(expectedRate(values.collaterals[ilk].pct), yearlyYield(rates.rates(values.collaterals[ilk].pct / 10))) <= TOLERANCE);
+        assertTrue(diffCalc(expectedRate(values.collaterals[ilk].pct * 10), yearlyYield(rates.rates(values.collaterals[ilk].pct))) <= TOLERANCE);
```

Comment:
same comment as other PR...basis points all the way down would be nice, but again no need to block on it if you disagree

Reply:
I was just making the smallest incision here. I can look into taking it the rest of the way.

Reply:
Sorry, looks like I didn't communicate clearly enough--I just meant that it was very inelegent to be multiplying `pct` by 10 here, so it would be nice if `expectedRate` could just expect a basis point argument, I didn't mean to use basis points to express all values.

Reply:
That's fine. Pushed a fix for this. The other changes are big improvements IMO since they are easier to inspect and reason about now and provide some addition protection against transcription errors.